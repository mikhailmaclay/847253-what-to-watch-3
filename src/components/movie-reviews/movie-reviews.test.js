// Libraries
import React from 'react';
import renderer from 'react-test-renderer';
// Components
import MovieReviews from './movie-reviews';

const reviews = [
  {
    id: 1,
    movie: 1,
    user: 1,
    date: `March 13, 2014 03:20:21`,
    rating: 10,
    text: `Hello world!`
  }
];

const users = [
  {id: 1, fullName: `Shia LaBeouf`}
];

it(`<MovieReviews/> should render correctly`, () => {
  const result = renderer.create(
      <MovieReviews reviews={reviews} users={users}/>
  ).toJSON();

  expect(result).toMatchSnapshot();
});
