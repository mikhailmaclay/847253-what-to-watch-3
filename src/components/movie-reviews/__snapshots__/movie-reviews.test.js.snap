// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MovieReviews/> should render correctly 1`] = `
<div
  className="movie-card__reviews movie-card__row"
>
  <div
    className="movie-card__reviews-col"
  >
    <div
      className="review"
    >
      <blockquote
        className="review__quote"
      >
        <p
          className="review__text"
        >
          Hello world!
        </p>
        <footer
          className="review__details"
        >
          <cite
            className="review__author"
          >
            Kate Muir
          </cite>
          <time
            className="review__date"
            dateTime="March 13, 2014 03:20:21"
          >
            March 13, 2014
          </time>
        </footer>
      </blockquote>
      <div
        className="review__rating"
      >
        10.0
      </div>
    </div>
  </div>
</div>
`;
